config/zones/GR.yaml

```diff
@@ -171,13 +177,19 @@ emissionFactors:
         source: EU-ETS, ENTSO-E 2022; Oberschelp, Christopher, et al. "Global emission
           hotspots of coal power generation."
         value: 1537.247562
+      - datetime: '2023-01-01'
+        source: EU-ETS, ENTSO-E 2023; IPCC 2014
+        value: 1278.46
```

Comment:
Emissions for the power plant _ΑΤΜΟΗΛΕΚΤΡΙΚΟΣ ΣΤΑΘΜΟΣ ΑΗΣ ΑΓΙΟΥ ΔΗΜΗΤΡΙΟΥ_ has been reduced drastically (5,867,033 in 2022 and 3,349,960 in 2023) while generation has increased from 3,784,599 to 4,026,773 MWh in 2023. Mapping of power plants is also improved.